builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakUserChecker.java

```diff
@@ -39,7 +40,7 @@
     private static final String ENDPOINT = "http://che-host:8080/api/token/user";
 
     @Inject
-    private KeycloakHttpJsonRequestFactory requestFactory;
+    private DefaultHttpJsonRequestFactory requestFactory;
```

Comment:
We explicitely use `DefaultHttpJsonRequestFactory` because we don't want to use the `KeycloakHttpJsonRequestFactory` that is bound into Guice by the fabric8 assembly. We don't want to secure the related request through Keycloak since this request is to retrieve Keycloak settings.